packages/core/src/accessanalyzer/terraformConfig/default.yaml

```diff
@@ -0,0 +1,121 @@
+# generate fake ARN
+# default can be specified using the following format:
+# {<key>?<default>}
+arnServiceMap:
+    aws_iam_policy: name?fakename
+    aws_iam_user_policy: name?fakename
+    aws_iam_role: name?fakename
+    aws_iam_role_policy: name
+    aws_iam_group_policy: name
+    aws_api_gateway_rest_api_policy: rest_api_id?fakeRestApiId
+    aws_backup_vault_policy: backup_vault_name?fakeBackupVaultName
+    aws_cloudwatch_event_bus_policy: event_bus_name?fakeEventBusName
+    aws_cloudwatch_log_destination_policy: destination_name?fakeDestinationName
+    aws_codeartifact_domain_permissions_policy: domain?fakeDomain
+    aws_codeartifact_repository_permissions_policy: repository?fakeRepository
+    aws_codebuild_resource_policy: fakename
+    aws_ecr_registry_policy: fakename
+    aws_ecr_repository_policy: repository?fakeRepositoryName
+    aws_ecrpublic_repository_policy: repository_name?fakeRepositoryName
+    aws_efs_file_system_policy: file_system_id?fakeFileSystemId
+    aws_elasticsearch_domain: domain_name?fakeDomainName
+    aws_elasticsearch_domain_policy: domain_name?fakeDomainName
+    aws_glacier_vault: name?fakename
+    aws_glacier_vault_lock: vault_name?fakeVaultName
+    aws_glue_resource_policy: fakeName
+    aws_iot_policy: name?fakename
+    aws_kms_external_key: fakeName
+    aws_kms_key: fakeName
+    aws_kms_replica_external_key: fakeName
+    aws_kms_replica_key: fakeName
+    # aws_lambda_layer_version_permission: layer_name?fakeLayberName
+    aws_media_store_container_policy: container_name?fakeContainerName
+    aws_networkfirewall_resource_policy: resource_arn?fakeResourceArn
+    aws_organizations_policy: name?fakename
+    aws_s3_access_point: name?fakename
+    aws_s3_bucket: bucket?fakeBucket
+    aws_s3_bucket_policy: bucket?fakeBucket
+    aws_s3control_access_point_policy: access_point_arn?fakeAccessPointArn
+    aws_s3control_bucket_policy: bucket?fakeBucket
+    aws_s3control_multi_region_access_point_policy: details.name?fakename
+    aws_s3control_object_lambda_access_point_policy: name?fakename
+    aws_ses_identity_policy: name?fakename
+    aws_sns_topic: name?fakename
+    aws_sns_topic_policy: arn?fakename
+    aws_sqs_queue: name?fakename
+    aws_sqs_queue_policy: fakeQueueUrl
+    aws_ssoadmin_permission_set_inline_policy: instance_arn?fakeSSOInstanceArn
+    aws_sagemaker_model_package_group_policy: model_package_group_name?fakeModelPackageGroupName
+    aws_secretsmanager_secret: name?fakename
+    aws_secretsmanager_secret_policy: secret_arn?fakeSecretArn
+    aws_transfer_access: server_id?fakeServerId
+    aws_transfer_user: user_name?fakeUserName
+    aws_vpc_endpoint: fakeName
+# iamChecks:
+#   - AccessAnalyzer
+
+# iamExceptions:
```

Comment:
Q: what are these?

Reply:
This file is needed for the Terraform CLI tool as a parameter, and is copied from the repo as well. We are copying it and providing it in the extension by default.

Reply:
i meant what are the commented ones

Reply:
I am not exactly sure, as it is pulled directly from the CLI tool.